.github/CODEOWNERS

```
@@ -20,11 +20,9 @@
 
 # CODEOWNERS
 
-*			@AgnostiqHQ/aq-engineers
-
 # Modifying these requires additional approval
 Dockerfile		@AgnostiqHQ/aq-managers
-.github/workflows/**	@AgnostiqHQ/aq-managers
+.github/**		@AgnostiqHQ/aq-managers
```

```suggestion
.github/**		@AgnostiqHQ/aq-managers
.pre-commit-config.yaml		@AgnostiqHQ/aq-managers
.prettierrc		@AgnostiqHQ/aq-managers
.pylintrc		@AgnostiqHQ/aq-managers
```

should we include these as well?

---

mostly this is for security reasons so I'm ok with people contributing to those other files